ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -86,12 +86,24 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   const visual = merge({}, DEFAULT_VISUAL, props.spec.visual);
 
   // convert Perses dashboard format to be ECharts compatible
-  const yAxis = {
+  const yAxis: YAXisComponentOption = {
     show: y_axis?.show ?? DEFAULT_Y_AXIS.show,
-    min: y_axis?.min, // leaves min and max undefined by default to let ECharts calcualate
+    min: y_axis?.min,
     max: y_axis?.max,
   };
 
+  if (yAxis.min === undefined) {
+    // sets minimum axis label relative to data instead of zero
+    yAxis.min = (value) => {
+      // https://echarts.apache.org/en/option.html#yAxis.min
+      if (value.min <= 1) {
+        return 0;
+      }
+      // allows for padding between origin and first series
+      return value.min * 0.8;
```

Comment:
Picked `0.8` arbitrarily to give just enough padding, but we'll probably need to tweak this

Reply:
Could you add that context to the comment, so future us knows this isn't some special number, and it's safe to tweak.

Reply:
Could be nice to make a named constant too? For easier tweaking + knowing what it means